files/Trimble_Quadri/Test_IFC44_Quadri.ifc

```diff
@@ -22,634 +22,370 @@ DATA;
718, 28.9074807672466), (506.653471707206, 589.054650638718, 28.8957717787119), (506.927149091003, 590.094162571477, 28.9263909300142), (507.248340754057, 589.856122866739, 28.9148597467525), (507.520535839853, 590.896057197358, 28.9453010799104), (507.842070505532, 590.658475395059, 28.9339477048918), (508.112805792247, 591.698823987273, 28.964211216935), (508.43467753171, 591.461693420075, 28.95303565313), (508.703975548779, 592.502448233543, 28.9831213410881), (509.026178462605, 592.265762210358, 28.972123591467), (509.294061769338, 593.306915299501, 29.0020314523697), (509.616589987141, 593.070667106891, 28.9912115199029), (509.883081172025, 594.11221062066, 29.0209415507798), (510.205928852025, 593.876393522136, 29.0102994384376), (510.471050532113, 594.918319700984, 29.0398516363183), (510.794211860732, 594.682926938171, 29.0293873470711), (511.057986680942, 595.725228113355, 29.0587617089853), (511.381455872353, 595.490252905991, 29.0484752458034), (511.643906504934, 596.532921497943, 29.0776717687808), (511.967677800683, 596.298357044812, 29.0675631346346), (512.228826944527, 597.341385561042, 29.0965818157048), (512.552894613036, 597.107225040207, 29.0866510135646), (512.812764993083, 598.150606075069, 29.1154918497573), (513.137123329288, 597.916842644103, 29.1057388825934), (513.395737695901, 598.960568876704, 29.1344018709382), (513.720381020787, 598.727195672691, 29.124826741721), (513.977762149239, 599.771259865491, 29.1533118792476), (514.302684809416, 599.538270006189, 29.1439145909475), (514.558855499257, 600.582665002439, 29.1722218746855), (514.884051866524, 600.350051586982, 29.1630024302728), (515.139034940948, 601.394770311657, 29.1911318572518), (515.464499411857, 601.162526419386, 29.182090259697), (515.718317717241, 602.207561875228, 29.2100418269467), (516.044044712646, 601.975680568023, 29.20117807922), (516.296721117993, 603.021025835304, 29.22895178377), (516.622705082613, 602.78950015665, 29.2202658888417), (516.874262478887, 603.835148391081, 29.2478617277218), (517.200497880869, 603.603971367003, 29.2393536885624), (517.450959180598, 604.649915799033, 29.2667716588021), (517.777440511054, 604.419080438325, 29.2584414783818), (518.026828647708, 605.465314371046, 29.2856815770108), (518.353550420303, 605.234813665505, 29.2775292583001), (518.601888347825, 606.281330473023, 29.3045914823481), (518.928845098271, 606.051157398615, 29.2966170283172), (519.176155790541, 607.097950525349, 29.3235013748138), (519.503342076176, 606.868098041508, 29.3157047884332), (519.7496485265, 607.915160999168, 29.342411254408), (520.077058925861, 607.685622049961, 29.334792538648), (520.322384146479, 608.732948418008, 29.3613211211306), (520.650013258783, 608.503715931904, 29.3538802789616), (520.894380280355, 609.55129935476, 29.3802309749818), (521.222222725119, 609.322366245789, 29.372968009374), (521.465654596323, 610.370200430509, 29.3991408159614), (521.793705012809, 610.141559598036, 29.3920557298853), (522.03622479974, 611.18963
```

Comment:
@larswik @SergejMuhic For `#229` and `#237` (and a lot of other lines in the file): related to https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/pull/120#discussion_r1179405350

Reply:
(Nope, still not working. Hmm.)